tests/integration/helm_default_service_monitors_test.go

```diff
@@ -0,0 +1,157 @@
+			require.Equal(t, "true", service.Labels[scrapeLabel])
+			require.Equal(t, "instrumentation", service.Labels[componentLabel])
+			return ctx
+		}).
+		Feature()
+
+	featPrometheusOperatedServiceMonitorLabels := features.New("instrumentation services contain scrape labels").
+		Assess("otelagent service scrape labels are present", func(ctx context.Context, t *testing.T, envConf *envconf.Config) context.Context {
+			name := "prometheus-operated"
+			service := terrak8s.GetService(t, ctxopts.KubectlOptions(ctx), name)
+			require.Equal(t, "true", service.Labels["operated-prometheus"])
+			return ctx
+		}).
+		Feature()
+
+	featEventsServiceMonitorLabels := features.New("otelcol-events service contain scrape labels").
+		Assess("otelcol-events service scrape labels are present", func(ctx context.Context, t *testing.T, envConf *envconf.Config) context.Context {
+			name := fmt.Sprintf("%s-sumologic-otelcol-events", ctxopts.HelmRelease(ctx))
+			service := terrak8s.GetService(t, ctxopts.KubectlOptions(ctx), name)
+			require.Equal(t, "otelcol-events", service.Labels[appLabel])
+			require.Equal(t, "true", service.Labels[scrapeLabel])
+			return ctx
+		}).
+		Feature()
+
+	featLogsServiceMonitorLabels := features.New("otelcol-logs-collector service contain scrape labels").
+		Assess("otelcol-logs-collector service scrape labels are present", func(ctx context.Context, t *testing.T, envConf *envconf.Config) context.Context {
+			name := fmt.Sprintf("%s-sumologic-otelcol-logs-collector", ctxopts.HelmRelease(ctx))
+			service := terrak8s.GetService(t, ctxopts.KubectlOptions(ctx), name)
+			require.Equal(t, "otelcol-logs-collector", service.Labels[appLabel])
+			require.Equal(t, "true", service.Labels[scrapeLabel])
+			return ctx
+		}).
+		Feature()
+
+	featMetadataServiceMonitorLabels := features.New("metadata-metrics service contain scrape labels").
+		Assess("metadata-metrics service scrape labels are present", func(ctx context.Context, t *testing.T, envConf *envconf.Config) context.Context {
+			name := fmt.Sprintf("%s-sumologic-metadata-metrics", ctxopts.HelmRelease(ctx))
+			service := terrak8s.GetService(t, ctxopts.KubectlOptions(ctx), name)
+			require.Equal(t, "fluentd-metrics", service.Labels[appLabel])
+			require.Equal(t, "true", service.Labels[scrapeLabel])
+			return ctx
+		}).
+		Assess("metadata-logs service scrape labels are present", func(ctx context.Context, t *testing.T, envConf *envconf.Config) context.Context {
+			name := fmt.Sprintf("%s-sumologic-metadata-logs", ctxopts.HelmRelease(ctx))
+			service := terrak8s.GetService(t, ctxopts.KubectlOptions(ctx), name)
+			require.Equal(t, "fluentd-logs", service.Labels[appLabel], "scrape label to equal true")
+			require.Equal(t, "true", service.Labels[scrapeLabel], "scrape label to be true")
+			return ctx
+		}).
+		Feature()
+
+	testenv.Test(t, featServiceMonitors, featServiceMonitorsSelectors, featInstrumentationServiceMonitorsLabels, featPrometheusOperatedServiceMonitorLabels,
```

Comment:
What I'd do with this is move all of these to template tests, and check in the default metrics test whether the metrics we expect from collection monitoring actually exist. In integration tests, we should check if we get the right data at the end, without worrying about how that is accomplished.